docs/Basics.md

```diff
@@ -7,71 +7,91 @@
+```
+npm install exoframe -g
+```
 
-This list can be extended via deployment templates.  
-You can find the list of available templates [on npm](https://www.npmjs.com/search?q=exoframe-template).  
-Templates can be installed by executing `exoframe template` command and entering complete template package name.
+Make sure you have [Exoframe server](https://github.com/exoframejs/exoframe-server) deployed, set it as your endpoint using:
 
-## Complex recipes
+```
+exoframe endpoint http://you.server.url
+```
 
-Exoframe also provides support for third-party complex deployment recipes.  
-They allow to quickly and easily deploy complex projects.
+Then login using:
 
-For example, you can deploy Wordpress with PHPMyAdmin by simply executing `exoframe setup` and entering [exoframe-recipe-wordpress](https://github.com/exoframejs/exoframe-recipe-wordpress) as desired recipe.
+```
+exoframe login
+```
 
-You can find the list of available recipes [on npm](https://www.npmjs.com/search?q=exoframe-recipe).
+Then you will be able to deploy your projects by simply running:
 
-## Commands
+```
+exoframe
+```
 
-| Command              | Description                                                          |
-| -------------------- | -------------------------------------------------------------------- |
-| deploy [path]        | Deploy specified path                                                |
-| config               | Generate or update project config for current path                   |
-| list                 | List currently deployed projects                                     |
-| rm <id>              | Remove existing deployment or project                                |
-| log <id>             | Get logs for existing deployment or project                          |
-| template [ls, rm]    | Add, list or remove deployment templates from the server             |
-| setup [recipe]       | Setup a complex recipe deployment                                    |
-| token [ls, rm]       | Generate, list or remove deployment tokens                           |
-| secret [new, ls, rm] | Create, list or remove deployment secrets                            |
-| login                | Login into Exoframe server                                           |
-| endpoint [url]       | Selects or adds the endpoint of Exoframe server                      |
-| rm-endpoint [url]    | Removes an existing endpoint of Exoframe server                      |
-| update [target]      | Gets current versions or updates given target (server, traefik, all) |
-| completion           | Generates bash completion script                                     |
+## Deployment
```

Comment:
I don't think this section is necessary here since it effectively repeats the last bit of previous section 🤔